doc/user/interop.md

```diff
@@ -287,6 +306,10 @@ order.
 
 `Truffle::Interop.read(object, name, value)`
 
+### `REMOVE`
+
+`Truffle::Interop.remove(object, name)`
+
```

Comment:
You need a blank entry under 'How to send messages using idiomatic Ruby' as well.